photonai_neuro/neuro_branch.py

```diff
@@ -49,6 +49,10 @@ def __iadd__(self, pipe_element: PipelineElement):
             self
 
         """
+        if pipe_element.name not in NeuroBranch.NEURO_ELEMENTS:
```

Comment:
Confusing if constructs

Reply:
This is not possible otherwise. It is the same solution as used in the photon registry.

Any idea for a better solution would be appreciated.

Reply:
See https://github.com/wwu-mmll/photonai/blob/fa57aaba6fe4dedc3233c9ecd4a1df255782fd31/photonai/base/photon_elements.py#L90